src/main/java/net/commuty/parking/model/AccessLog.java

```diff
@@ -40,7 +42,8 @@ public class AccessLog {
               @JsonProperty("granted") boolean granted,
               @JsonProperty("identificationMethod") String identificationMethod,
               @JsonProperty("identificationValue") String identificationValue,
-              @JsonProperty("reason") String reason) {
+              @JsonProperty("reason") String reason,
+              @JsonProperty("attributes") Object attributes) {
```

Comment:
`Object` is too permissive considering you actually want to force (at least) a `Map<String, Object>`